<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1481</width>
    <height>949</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1481</width>
      <height>841</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="inputTab">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QWidget" name="rosCoreWidget" native="true">
      <property name="geometry">
       <rect>
        <x>859</x>
        <y>20</y>
        <width>541</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="rosPlayWidget" native="true">
      <property name="geometry">
       <rect>
        <x>859</x>
        <y>400</y>
        <width>541</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="rawVideo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>601</width>
        <height>700</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>20</width>
        <height>791</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="outputTab">
     <attribute name="title">
      <string>Output</string>
     </attribute>
     <widget class="QLabel" name="processedVideo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>581</width>
        <height>700</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="visualisedVideo">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>50</y>
        <width>601</width>
        <height>700</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>10</y>
        <width>20</width>
        <height>791</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>860</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1481</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="saveMenuButton"/>
    <addaction name="quitMenuButton"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="saveMenuButton">
   <property name="text">
    <string>Write to bag and quit</string>
   </property>
  </action>
  <action name="quitMenuButton">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>

